index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="styles.css">
+</head>
+
+<body>
+    <!-- Start of the header -->
+    <header>
+        <div class="center">
+        <nav>
+            <ul>
+                <li><span><a href="./about.html" >About Me</a></span></li>
+                <li><span><a href="./portfolio.html">Soriano's Portfolio</a></span></li>
+                <!-- <li><span><a href="mailto:dsorin@proton.me" class="button">Contact</a></span></li> -->
+            </ul>
+        </nav>
+    </div>
+</header>
+    <<body>
+        <div class="waviy">
+          <span style="--i:1">D</span>
+          <span style="--i:2">.</span>
+          <span style="--i:3">S</span>
+          <span style="--i:4">O</span>
+          <span style="--i:5">R</span>
```

Comment:
This is a cool animation you're doing, but I'd look for an approach that doesn't require passing variables to CSS via inline style tags.